### ISSUE TITLE
nginx stops at the first start after loading docker desktop

### ISSUE THREAD

**Issue by 5nap5can** [boschkundendienst/guacamole-docker-compose#102]
After booting the docker desktop, the containers start automatically. However, Nginx stops with an exited (127) error. If I manually start it, it starts already. I manually stop and start the whole container at any time, it runs without any problems. However, you have to intervene manually just after the system boot. For this reason, the machine running docker cannot be accessed via https after every power outage or other shutdown.
[nginx.txt](https://github.com/user-attachments/files/17090640/nginx.txt)

**Comment by boschkundendienst** [comment]
First of all, nowhere within this project docker desktop is used at all. Please understand that I can not support user specific setups. 5 minutes ago I tested on a Linux machine with Docker version 27.1.2 and it worked flawlessly.
I can also see, that you run everything in Microsofts Windows Subsystem for Linux. So you want me to troubleshoot a project designed to be run on Linux but you run it on Windows? - No way.

Besides that, you should verify, that all ports the docker containers try to open are not used by any other program/service on your windows box. Also networking in WSL can be a little bit tricky, too.
The only thing I can see from your logs is a `SIGQUIT` received and after that a graceful shutdown so nothing to worry.

- Did you try to use `reset.sh` and afterwards `prepare.sh` to start from scratch?
- Is your user in the group `docker` in WSL?
- Is your user allowed to open ports from WSL via docker on your windows machine?
- Is the port `8443` (nginx) unused/free before you start the container?
- Did you open port 8443 in the **Windows Firewall** manually? Make sure it is open!

Anyways I would not run Guacamole in Docker Desktop on Windows. The project is demonstration of Guacamole and not designed to be used in production. If you still have problems, please follow the [official Guacamole documentation](https://guacamole.apache.org/doc/gug/guacamole-docker.html) and maybe also take a look at the [setup of docker with WSL2](https://docs.docker.com/desktop/wsl/).

Kind regards, Peter

**Comment by 5nap5can** [comment]
Thank you for your answer. Since I have also experienced this with other containers, the docker desktop is probably the key to the error.
So many comments on this project that it works well. Unfortunately, due to the self-signed certificate, it cannot be used in any meaningful way. I found another repository in which the Let's Encrypt certificate manager is built into the container, it can really be used on any machine. The launch issues still exist there, but it is already in the process of being fixed. Regardless, it was a very good learning project, thank you for your work.
